soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3691,9 +3691,12 @@ void DrawRandoEditor(bool& open) {
                         SohImGui::EnhancementCombobox("gRandomizeGanonTrial", randoGanonsTrial, 2, 0);
                         if (CVar_GetS32("gRandomizeGanonTrial", 0) == 0) {
                         */
-                            SohImGui::EnhancementSliderInt("Ganon's Trial Count: %d", "##RandoTrialCount",
-                                                           "gRandomizeGanonTrialCount", 0, 6, "");
-                            InsertHelpHoverText("Set the number of trials required to enter\nGanon's Tower.");
+                            //SohImGui::EnhancementSliderInt("Ganon's Trial Count: %d", "##RandoTrialCount",
+                            //                               "gRandomizeGanonTrialCount", 0, 6, "");
+                            //InsertHelpHoverText("Set the number of trials required to enter\nGanon's Tower.");
+                        SohImGui::EnhancementCheckbox("Skip Ganon's Trials?", "gRandomizeGanonTrialCount");
```

Comment:
tiny nitpick, but i don't think we want the `?` in `Skip Ganon's Trials` here, since there will be a `?` added by `InsertHelpHoverText`